README.md

```diff
@@ -211,6 +211,17 @@ web_ami:
 
 Note that the corresponding array resolver is named `latest_amis_by_tags`
 
+### Latest AMI by attribute
+
+Looks up the latest AMI ID by a given set of attributes. This selects the latest wily hvm AMI from Ubuntu (using the account id):
+
+```yaml
+bastion_ami:
+  latest_amis: owner_id=099720109477,name=ubuntu/images/hvm/ubuntu-wily-15.10-amd64-server-*
```

Comment:
It might be worth linking to the documentation to find out other things to filter by - `https://docs.aws.amazon.com/sdkforruby/api/Aws/EC2/Client.html#describe_images-instance_method`.